oeis.py

```diff
@@ -168,6 +184,19 @@ def phi(n: int) -> int:
     return [phi(x) for x in range(start, start + limit)]
 
 
+@oeis
+def A023811(start: int = 0, limit: int = 20) -> Collection[int]:
+    def largest_metadrome(n):
+        result = 0
+        for i,j in enumerate(range(n-2,-1, -1),start=1):
```

Comment:
This is not PEP8 compliant.
```suggestion
        for i, j in enumerate(range(n - 2, -1, -1), start=1):
```

But you should just use black (add it as a plugin in your IDE).